src/components/Clock/styles.ts

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+
+export const ClockContainer = styled.div`
+  background-color: #cb6e6e;
+  display: grid;
+  justify-content: center;
+  border-radius: 5px;
+  width: 30rem;
```

Comment:
[fix] You should use `max-width` instead of `width` when you want to set a limit to how wide an element can grow. If you use `width`, it's not going to shrink when the page's width gets smaller than the defined `width` value.

Reply:
[suggestion] You can add a `margin: 1rem auto` here to center the element.

Reply:
[suggestion] Whenever you need to center a block element that has a defined width, you can use `margin-left: auto` and `margin-right: auto`, or just `margin: {whatever vertical margin you want} auto;